app/test_main.py

```diff
@@ -1,3 +1,53 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "dog_age,cat_age,expected_result",
+    [
+        pytest.param(
```

Comment:
There is no need to use `pytest.param` with `id` here, omit it here, and for error tests.